6.3.long_cat_is_long.py

```diff
@@ -0,0 +1,18 @@
+import string
+
+text = """
+You see, wire telegraph is a kind of a very, very long cat.
+You pull his tail in New York and his head is meowing in Los Angeles.
+Do you understand this?
+And radio operates exactly the same way: you send signals here, they receive them there.
+The only difference is that there is no cat.
+"""
+
+
+def count_words(txt):
```

Comment:
rename 'txt' to 'text'
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Reply:
fixed it